plugins/kn-migration/README.md

```diff
@@ -0,0 +1,66 @@
+# kn migration migrate
+
+Migrate Knative services from source cluster to destination cluster
+
+## Plugin introduction
+### Synopsis
+
+Migrate Knative services from source cluster to destination cluster
+
+```
+> kn migration --help
+Migrate Knative services from source cluster to destination cluster
```

Comment:
Be consistent. Remove extra ">" and spaces in formatting.

Reply:
fixed in the second commit: https://github.com/knative/client-contrib/pull/6/commits/e77b98b6cebdccc7a81fa81f9c508488611a2d3b